src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const minLongTerm = 7;
+  const minLowTerm = 3;
+
+  if (days >= minLongTerm) {
+    return days * 40 - 50;
+  }
+
+  if (days >= minLowTerm) {
+    return days * 40 - 20;
+  }
+
+  if (days < minLowTerm) {
+    return days * 40;
+  }
```

Comment:
The condition `if (days < minLowTerm)` is redundant because if the previous conditions are not met, it is implied that `days` is less than `minLowTerm`. You can replace this condition with an `else` statement for better readability.